### PR TITLE
enhancement/issue 400 switch to internal fonts

### LINKED ISSUE #400: site uses external fonts, should use internal fonts instead
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [x] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary

The greenwood site uses external fonts. We should download and use the fonts internally and prevent any additional external source from slowing down the initial page load. This may improve performance with this small optimization.

## Details

Fonts within `theme.css` and `components/shelf/shelf.css` could instead be fetched internally.  Google fonts can be downloaded and hosted.

### PR DESCRIPTION
**Pull request by hutchgrant**
## Related Issue
Resolve #400

## Summary of Changes

* switch theme.css import to use local font
* switch shelf/shelf.css import to use local font

### CODE REVIEW

**Comment by hutchgrant** [comment]
yea google didn't provide woff2 fonts in the "download font family" link. I'll have to do it another way.

**Comment by hutchgrant** [comment]
With woff2 internal font I saw and 8% performance gain in lighthouse test score locally.

File: www/assets/fonts/SourceSansPro.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: url('/assets/fonts/SourceSansPro/SourceSansPro-Regular.ttf') format('truetype');
```
Comment (thescientist13) [inline www/assets/fonts/SourceSansPro.css:7]: still seeing a _.ttf_ here?

**Comment by hutchgrant** [comment]
I'll do one better we can support IE6 - IE8 https://google-webfonts-helper.herokuapp.com/fonts/source-sans-pro?subsets=latin

**Comment by hutchgrant** [comment]
By adding these additional charsets and styles, you'll notice the fonts appear bolder. I think this is a fix rather than a bug. Our heading elements have a font weight which wasn't reflected correctly by original font. The downside is the fonts are larger.

**Comment by hutchgrant** [comment]
Okay woff2, woff, and ttf  for android,safari and ios.

**Comment by hutchgrant** [comment]
Definetly

**Comment by hutchgrant** [comment]
Yes we can do that here as well.

**Pull request merged**